client-internal/src/test/resources/terracotta/10/tc-config-big.xml

```diff
@@ -40,16 +40,16 @@
 
   <servers>
     <server host="localhost" name="Server1">
-      <tsa-port>9410</tsa-port>
-      <tsa-group-port>9430</tsa-group-port>
+<!--      <tsa-port>9410</tsa-port>-->
+<!--      <tsa-group-port>9430</tsa-group-port>-->
```

Comment:
No need to specify any port at all now in the XML files. If set, they will be picked up. Otherwise a random port will be allocated